lib/LicenseTermsList/LicenseTermsList.js

```diff
@@ -19,9 +19,18 @@ export default class LicenseTermsList extends React.Component {
       label: PropTypes.string,
       name: PropTypes.string,
       primary: PropTypes.bool,
+      defaultInternal: PropTypes.bool,
     })),
   };
 
+  renderVisibility = (internal, term) => {
+    if ((internal === false) || (!internal && term.defaultInternal === false)) {
```

Comment:
Shortcutting with `!internal` is dangerous here since `internal` can be `false` as well as `undefined`. Additionally, the intent of this line could be difficult to understand for a new developer who doesn't have the full background of the various cases involved in calculating term visibility.